src/module.c

```diff
@@ -7189,199 +7215,197 @@ void ModuleForkDoneHandler(int exitcode, int bysignal) {
  *
  * * RedisModuleEvent_ReplicationRoleChanged:
  *
- *   This event is called when the instance switches from master
- *   to replica or the other way around, however the event is
- *   also called when the replica remains a replica but starts to
- *   replicate with a different master.
+ *     This event is called when the instance switches from master
+ *     to replica or the other way around, however the event is
+ *     also called when the replica remains a replica but starts to
+ *     replicate with a different master.
```

Comment:
unlike the previous one i commented on (which looked good in redis.io), this one does indeed looks messed up in redis.io.
i wonder what's the difference from the previous one i commented on..
maybe it's the empty line which we should just delete instead of adding indentation in both of these?

Reply:
It's the indentation. The body of multi-paragraph list items need 4 spaces indentation. The empty lines between the bullets don't matter. I can revert the empty lines to minimize the diff.